meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -360,6 +360,35 @@ BOOTSCR_LOAD_ADDR_imx8mm-lpddr4-evk = "0x44800000"
 SOTA_CLIENT_FEATURES_remove_sota_imx8mm-lpddr4-evk-sec = "ubootenv"
 SOTA_EXTRA_CLIENT_FEATURES_append_sota_imx8mm-lpddr4-evk-sec = " fiovb"
 
+# iMX8MP
+UBOOT_SIGN_ENABLE_sota_mx8mp = "1"
+IMX_DEFAULT_BOOTLOADER_mx8mp ?= "u-boot-fio"
+UBOOT_DTB_LOADADDRESS_mx8mp = "0x43000000"
+PREFERRED_PROVIDER_virtual/trusted-firmware-a_mx8mp ?= "imx-atf"
+EXTRA_IMAGEDEPENDS_append_mx8mp = "virtual/trusted-firmware-a"
+UBOOT_CLASSES_mx8mp = "uboot-fitimage"
+ATF_BINARY_mx8mp = "arm-trusted-firmware.bin"
+UBOOT_CONFIG_mx8mp = ""
+LMP_BOOT_FIRMWARE_FILES_mx8mp = "imx-boot u-boot.itb"
+IMAGE_BOOT_FILES_mx8mp = "imx-boot u-boot.itb ${KERNEL_IMAGETYPE} ${@make_dtb_boot_files(d)} boot.scr uEnv.txt"
+IMAGE_BOOT_FILES_sota_mx8mp = "boot.itb"
+WKS_FILE_DEPENDS_append_mx8mp = " u-boot-default-script"
+PREFERRED_PROVIDER_u-boot-default-script_mx8mp = "u-boot-base-scr"
+PREFERRED_PROVIDER_u-boot-default-script_sota_mx8mp = "u-boot-ostree-scr-fit"
+KERNEL_IMAGETYPE_sota_mx8mp = "fitImage"
+KERNEL_CLASSES_sota_mx8mp = " kernel-lmp-fitimage "
+## iMX8: Use latest NXP BSP downstream kernel
+PREFERRED_PROVIDER_virtual/kernel_mx8mp ?= "linux-lmp-fslc-imx"
+MACHINE_FIRMWARE_mx8mp = "linux-firmware-imx-sdma-imx7d"
+WKS_FILE_sota_mx8mp = "sdimage-imx8-spl-sota.wks.in"
+IMXBOOT_TARGETS_mx8mp = "flash_evk_spl"
+## iMX8MP EVK
+OSTREE_KERNEL_ARGS_imx8mp-lpddr4-evk ?= "console=tty1 console=ttymxc1,115200 earlycon=ec_imx6q,0x30890000,115200 root=/dev/mmcblk2p2 rootfstype=ext4"
+SOTA_CLIENT_FEATURES_append_imx8mp-lpddr4-evk = " ubootenv"
+MACHINE_FEATURES_remove_imx8mp-lpddr4-evk = "qca9377 qca6174"
```

Comment:
Not sure if this line is needed, wasn't able to find these features in the machine conf.

Reply:
Yep, this looks like a copy pasta issue.  Removing.